EasyPost.Tests/FixtureData/Components/Addresses.cs

```diff
@@ -0,0 +1,23 @@
+using System.Collections.Generic;
```

Comment:
Thought: Do we need these all in separate files/does it make sense? If not, let's consolidate since it seems we aren't really using a separate namespace for fixture data and can reduce a bunch of the complexity by having all fixture definitions in one file like the other libs have. We don't at the end of the day have too many of them.

Reply:
We can certainly put them all in one file. I was just replicating Java, which by convention requires them to all be in separate files.

Reply:
Which of the two options feels most appropriate? I'm good for either.

Reply:
Combined, resolved.